src/citrine/informatics/descriptors.py

```diff
@@ -252,3 +253,13 @@ def __str__(self):
 
     def __repr__(self):
         return "FormulationDescriptor(key={})".format(self.key)
+
+    @staticmethod
+    def hierarchical() -> "FormulationDescriptor":
+        """The hierarchical formulation descriptor with key 'Formulation'."""
+        return FormulationDescriptor("Formulation")
+
+    @staticmethod
+    def flat() -> "FormulationDescriptor":
```

Comment:
Is this (using static methods to create specific instantiations of a class) Python idiomatic? It feels so much like Scala that it makes me wary. Maybe I'm over-thinking it

Reply:
Looks like the googles says we should use a `classmethod` for alternate constructors: https://realpython.com/python-multiple-constructors/#defining-multiple-class-constructors:~:text=Remove%20ads-,Providing%20Multiple%20Constructors%20With%20%40classmethod%20in%20Python,-A%20powerful%20technique